src/core/worker.js

```diff
@@ -436,6 +437,55 @@ var WorkerMessageHandler = PDFJS.WorkerMessageHandler = {
       });
     });
 
+    handler.on('GetRawObject', function wphGetRawObject(data) {
+      function clone(obj) {
+        if (!obj || isBool(obj) || isNum(obj) || isString(obj)) {
+          return obj;
+        } else if (isName(obj)) {
+          return obj;
```

Comment:
I accidentally deleted my previous comment, but I asked to return `{ name: obj.name };` to be explicit about its return value, so that whatever `GetRawObject` within the worker returns is also what you get on the main thread).